keyboards/ashpil/modelm_usbc/keymaps/ashpil/rules.mk

```diff
@@ -0,0 +1,34 @@
+# MCU name
```

Comment:
The only rules you need in this file are the ones that are different from those in `keyboards/ashpil/modelm_usbc/rules.mk`.

Reply:
Yep, this file could literally just be:
```make
COMBO_ENABLE = yes
```